04_HowTos/SPARCL/Plot_SPARCL_Spectra_with_Jdaviz.ipynb

```diff
@@ -9,8 +9,8 @@
    "source": [
```

Comment:
<p>Is this included here because the function is not in the 23.1 kernel? (If so, it could potentially be imported directly from <code>desiutil</code>  if we update to the next kernel)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/astro-datalab/notebooks-latest/pull/264/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>Yes, that's correct, though as you say, if desiname is already included with the DR1 data, we don't need a copy of the function.</p><p> </p>